forc/Cargo.toml

```
@@ -31,6 +31,7 @@ sway-fmt = { version = "0.2.1", path = "../sway-fmt" }
 sway-server = { version = "0.2.1", path = "../sway-server" }
 sway-utils = { version = "0.2.1", path = "../sway-utils" }
 sway-types = { version = "0.2.1", path = "../sway-types" }
+taplo = "0.7.0"
```

I know other deps don't all do this, but going forward the convention is usually to pin a minor, not patch, version.
```suggestion
taplo = "0.7"
```